src/test/java/org/fundationjala/at13/spaceinvaders/MenuTest.java

```diff
@@ -0,0 +1,23 @@
+package org.fundationjala.at13.spaceinvaders;
+import org.fundacionjala.at13.spaceinvaders.Menu;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+
+public class MenuTest {
+    
+    @Test
+    public void itDisplaySpaceByOptionOne() {
```

Comment:
This method said that "Displays the Space by option 1". But in Lin 17 the method said that Display by option 0.